src/fhirtypes/StructureDefinition.ts

```diff
@@ -82,6 +83,29 @@ export class StructureDefinition {
    */
   private _sdStructureDefinition: StructureDefinition;
 
+  private _validationErrors: ValidationError[];
+
+  get validationErrors(): ValidationError[] {
+    return this._validationErrors;
+  }
```

Comment:
I have the same thoughts/concerns as I noted in the `ElementDefinition` implementation.